Construction DX Idea (standalone).html

```
@@ -846,6 +846,21 @@
                     <sc-if value="{{ phaseView.phaseNote }}" hint-placeholder-val="{{ false }}">
                       <div style="font-size:11.5px;color:#8A97A8;">メモ: {{ phaseView.phaseNote }}</div>
                     </sc-if>
+                    <sc-if value="{{ phaseView.nextActionHint }}" hint-placeholder-val="{{ false }}">
+                      <div style="background:#EEF0FB;border-radius:6px;padding:8px 10px;font-size:11.5px;color:#3B2F91;">💡 次の必要Action: {{ phaseView.nextActionHint }}</div>
+                    </sc-if>
+                    <sc-if value="{{ phaseView.hasChecklist }}" hint-placeholder-val="{{ false }}">
+                      <div style="display:flex;flex-direction:column;gap:4px;">
+                        <div style="font-size:11.5px;font-weight:600;color:#5A6678;">必須成果物チェックリスト（{{ phaseView.checklistDoneCount }}/{{ phaseView.checklistTotalCount }}）</div>
+                        <sc-for list="{{ phaseView.checklistRows }}" as="ci" hint-placeholder-count="2">
+                          <label style="display:flex;align-items:center;gap:6px;font-size:11.5px;{{ ci.rowStyle }}">
+                            <input type="checkbox" checked="{{ ci.done }}" onChange="{{ ci.onToggle }}"></input>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </input> ]

(tag-pair)